src/commands.ts

```diff
@@ -0,0 +1,344 @@
+import {
+	LoadbgParameters,
+	LoadParameters,
+	PlayParameters,
+	PauseParameters,
+	ResumeParameters,
+	StopParameters,
+	ClearParameters,
+	CallParameters,
+	SwapParameters,
+	RemoveParameters,
+	PrintParameters,
+	LogLevelParameters,
+	LogCategoryParameters,
+	SetParameters,
+	LockParameters,
+	DataStoreParameters,
+	DataRetrieveParameters,
+	DataListParameters,
+	DataRemoveParameters,
+	CgAddParameters,
+	CgPlayParameters,
+	CgStopParameters,
+	CgNextParameters,
+	CgRemoveParameters,
+	CgClearParameters,
+	CgUpdateParameters,
+	CgInvokeParameters,
+	CgInfoParameters,
+	MixerKeyerParameters,
+	MixerChromaParameters,
+	MixerBlendParameters,
+	MixerInvertParameters,
+	MixerOpacityParameters,
+	MixerBrightnessParameters,
+	MixerSaturationParameters,
+	MixerContrastParameters,
+	MixerLevelsParameters,
+	MixerFillParameters,
+	MixerClipParameters,
+	MixerAnchorParameters,
+	MixerCropParameters,
+	MixerRotationParameters,
+	MixerPerspectiveParameters,
+	MixerMipmapParameters,
+	MixerVolumeParameters,
+	MixerMastervolumeParameters,
+	MixerStraightAlphaOutputParameters,
+	MixerGridParameters,
+	MixerCommitParameters,
+	MixerClearParameters,
+	ThumbnailListParameters,
+	ThumbnailRetrieveParameters,
+	ThumbnailGenerateParameters,
+	CinfParameters,
+	ClsParameters,
+	FlsParameters,
+	TlsParameters,
+	VersionParameters,
+	InfoParameters,
+	InfoTemplateParameters,
+	InfoConfigParameters,
+	InfoPathsParameters,
+	InfoSystemParameters,
+	InfoServerParameters,
+	InfoQueuesParameters,
+	InfoThreadsParameters,
+	InfoDelayParameters,
+	DiagParameters,
+	GlInfoParameters,
+	GlGcParameters,
+	ByeParameters,
+	KillParameters,
+	RestartParameters,
+	AddParameters,
+	ThumbnailGenerateAllParameters,
+	ChannelGridParameters,
+	PlayDecklinkParameters,
+	LoadbgDecklinkParameters,
+	LoadbgHtmlParameters,
+	LoadbgRouteParameters,
+	PlayHtmlParameters,
+	PlayRouteParameters,
+} from './parameters'
+
+export enum Commands {
```

Comment:
(Personal preference)
An enum should not have a plural name. `Commands.Play` looks wrong to me, `Command.Play` is better.